spec/root-apps/systemjs.spec.js

```diff
@@ -34,39 +35,57 @@ describe("SystemJS loader :", () => {
 
 			return ogSystemNormalize.call(this, name, ...rest);
 		}
+	});
 
-		resetSingleSpa()
-		.then(() => {
-			singleSpa.setLoader(SystemJS);
-			singleSpa.start();
-			done();
+	describe(`single-spa started :`, () => {
```

Comment:
I created a new suite called `single-spa started` so that we can do tests for when singleSpa.start() has been called, and another suit for when singleSpa.start() has not been called.